raiwidgets/raiwidgets/error_analysis_dashboard_input.py

```diff
@@ -346,6 +346,8 @@ def setup_local(self, explanation, model, dataset, true_y, classes,
                 metric = Metrics.ERROR_RATE
             else:
                 metric = self._error_analyzer.metric
+        elif self._error_analyzer.model_task == ModelTask.OBJECT_DETECTION:
```

Comment:
Hi @imatiach-msft , I haven't added any unit test for the OD case here as that'll require the `torch` and `torchvision` modules to load the fridge model, and hence may increase dependencies on this module (+ pass on similar torch version issues to this module in the future with 1.13 v/s >2.0).

Curious if you have suggestions to navigate this for purposes of unit testing, or if I shouldn't worry about testing here. Thanks!